cypress/integration/plp.test.js

```diff
@@ -63,11 +80,15 @@ describe('Search page Filters and Sorting options', () => {
   })
 
   it('Sort products by price_desc', () => {
+    cy.visit(pages.collection, options)
+    cy.waitForHydration()
+    const priceId = '[data-testid="store-card"] [data-testid="price"]'
```

Comment:
Set it as const, prevent change it multiple times in future.

Reply:
What do you mean by const? I'm confused because it's already using `const`.